schedulelambda/src/main/scala/com/gu/notificationschedule/external/Jackson.scala

```diff
@@ -0,0 +1,15 @@
+package com.gu.notificationschedule.external
+
+import com.fasterxml.jackson.databind.{ DeserializationFeature, ObjectMapper }
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
+
+object Jackson {
```

Comment:
I think this isn't used, and unless I'm mistaken, won't be used in the future so you can drop the jackson library and this file.

Reply:
Removed